src/components/inputs/ui-select-extended/ui-select-extended.tsx

```diff
@@ -45,7 +45,10 @@ const UISelectExtended: React.FC<OHRIFormFieldProps> = ({ question, handler, onC
   }, [encounterContext.sessionMode, question.readonly, question.inlineRendering, layoutType, workspaceLayout]);
 
   useEffect(() => {
-    const datasourceName = question.questionOptions?.datasource?.name;
+    let datasourceName = question.questionOptions?.datasource?.name;
+    if (question.type === 'encounterProvider') {
+      datasourceName = question.type;
+    }
```

Comment:
@kajambiya Any good reason for making this component aware of the `encounterProvider` type?

Reply:
Same reason as highlighted above. Ideally if a control doesn't explicitly define a datasource, it will search for the datasource using the rendering (which will be the same name provided in the `controlTemplates`) but the AMPATH case makes it unique since they use type and not rendering to specify `encounterLocation`